### PR TITLE
Add bokchoy tests

### PR DESCRIPTION
**Pull request by estute**

### CODE REVIEW

**Comment by michaelyoungstrom** [comment]
Adds in a bok-choy test to verify the main Jenkins config panel was setup properly.

**Comment by michaelyoungstrom** [comment]
@estute should also review... but this is his PR so I can't tag him.

**Comment by estute** [comment]
:+1: I can't approve something I am the author of. But LGTM once you squash the commits

File: .travis.yml [hunk 1]
```diff
@@ -21,12 +21,23 @@ cache:
         - $HOME/.gradle/wrapper/
 
 before_install:
+    # browser set up for e2e tests
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    - wget https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz
```
Comment (jzoldak) [inline .travis.yml]: nit: we might want to host this in a testeng aws account s3 bucket

**Pull request merged**